source/MaterialXCore/Document.h

```diff
@@ -170,6 +170,13 @@ class MX_CORE_API Document : public GraphElement
         removeChildOfType<GeomPropDef>(name);
     }
 
+    /// @}
+    /// @name Material Outputs
+    /// @{
+    
+    /// Return outputs on nodegraps which are of type "material"
```

Comment:
I would recommend phrasing this as "Return material-type outputs for all nodegraphs in the document."

Reply:
Will do.